framework/include/problems/EigenProblem.h

```diff
@@ -102,6 +107,11 @@ class EigenProblem : public FEProblemBase
    */
   void setEigenproblemType(Moose::EigenProblemType eigen_problem_type);
 
+  /**
+   * Override this for auto initialization
+   */
+  void computeUserObjects(const ExecFlagType & type, const Moose::AuxGroup & group) override;
+
```

Comment:
I can not make a connection between `auto-initialization` and `computeUserObjects`

Reply:
This is the piece of code in `FEProblemBase::initialSetup` for why:
```
  if (!_app.isRecovering())
  {
    computeUserObjects(EXEC_INITIAL, Moose::PRE_IC);

    for (THREAD_ID tid = 0; tid < n_threads; tid++)
      _ics.initialSetup(tid);
    _scalar_ics.initialSetup();

    CONSOLE_TIMED_PRINT("Projecting initial condition");
    projectSolution();
  }
```
I do not want to create a new virtual function just for this.

Reply:
Create a tiny function makes more sense that override `computeUserObjects` that has non connection to auto_initialization .

I feel like this even needs to happen before `computeUserObjects(EXEC_INITIAL, Moose::PRE_IC);`

Reply:
Sounds good. Is a name like `completeAutoInitialization` good or you can suggest a better name?

Reply:
It sounds fine for me